src/main.ts

```diff
@@ -312,18 +323,24 @@ export async function activate(
     if (!useOmnisharpServer) {
         tryGetCSharpDevKitExtensionExports(csharpLogObserver);
 
-        const languageServerExport = new RoslynLanguageServerExport(roslynLanguageServerPromise!);
+        // If we got here, the server should definitely have been created.
+        util.isNotNull(roslynLanguageServerStartedPromise);
```

Comment:
At this point these are unfulfilled promises, right? When would they be null?

Reply:
They can be null if useOmnisharpServer is true - but it should never be null if it was false, hence the assertion that its not null inside this if

Reply:
Note - this file could probably be refactored so this check didn't need to happen, but that would be a fairly large change and not worth doing in this PR.